velox/dwio/parquet/writer/Writer.cpp

```diff
@@ -29,7 +29,7 @@ class ArrowDataBufferSink : public arrow::io::OutputStream {
   ArrowDataBufferSink(
       std::unique_ptr<dwio::common::DataSink> sink,
       memory::MemoryPool& pool,
-      double growRatio = 1)
+      double growRatio = 1.5)
```

Comment:
Would you add a comment to explain this parameter and how the default value of 1.5 was chosen? 

BTW, is there any reason we seem to duplicate this default here and in Writer.h? Would it be possible to avoid duplication?

Reply:
The purpose of the growRatio in Writer.h is to provide configurability when writing data using the Parquet writer. This value is then passed to the ArrowDataBufferSink, which helps manage the growth of the buffer used during the writing process. It seems we can remove the default value in ArrowDataBufferSink.